src/snowflake/snowpark/functions.py

```diff
@@ -3356,6 +3356,19 @@ def array_max(array: ColumnOrName) -> Column:
     return builtin("array_max")(array)
 
 
+def array_flatten(array: ColumnOrName) -> Column:
+    """Returns a single array from an array or arrays. If the array is nested more than
+    two levels deep, then only a single level of nesting is removed.
+
+    Must enable parameter `ENABLE_ARRAY_FLATTEN_FUNCTION` in your session.
```

Comment:
Is this a public parameter that user can change?

Reply:
It is. We are waiting for [BCR](https://snowflakecomputing.atlassian.net/browse/BCR-1239) to go through so setting this param will no longer be necessary.